### ISSUE TITLE
Issues running on Raspberry pi 4

### ISSUE THREAD

**Issue by quicksilver7837** [ptitSeb/freespace2#2]
I was able to make the linux version for my raspberry pi 4 but it appears to be using desktop OpenGL so it requires running in a x window (which I am able to successfully although my mouse cursor is invisible in the menus, but the rest of the game appears to work). I know you didnt say you supported raspberry pi but curious if there is a way to configure it to make using OpenGL ES.

**Comment by ptitSeb** [comment]
Yes, add `HAVE_GLES=1` to your make command to have the GLES 1.1 backend.

You may also need to tweak `src/graphics/eglport.cpp` to have the right EGL context created.
